docs/stardoc.bzl

```diff
@@ -53,11 +53,11 @@ def stardoc(
         name,
         out,
         input,
-        aspect_template = "@rules_nixpkgs_core//docs:templates/aspect.vm",
-        func_template = "@rules_nixpkgs_core//docs:templates/func.vm",
-        header_template = "@rules_nixpkgs_core//docs:templates/header.vm",
-        provider_template = "@rules_nixpkgs_core//docs:templates/provider.vm",
-        rule_template = "@rules_nixpkgs_core//docs:templates/rule.vm",
+        aspect_template = "@io_tweag_rules_nixpkgs//docs:templates/aspect.vm",
```

Comment:
I think these should become references to the docs module, e.g. `@rules_nixpkgs_docs//:templates/rule.vm`.